docs/recipes/docker-server/update-dockerhub.sh

```diff
@@ -1,8 +1,7 @@
 #!/bin/bash
 
 CURRENT_VERSION=$(node -e "console.log(require('./package.json').dependencies['@lhci/server'])")
-# NEXT_VERSION=$(yarn info @lhci/server | grep 'latest:' -A 1 | tail -n 1 | grep -o "'.*'" | sed s/\'//g)
-NEXT_VERSION=0.3.3
+NEXT_VERSION=$(yarn info @lhci/server | grep 'latest:' -A 1 | tail -n 1 | grep -o "'.*'" | sed s/\'//g)
```

Comment:
woah. maybe do a call to node like the line above? this is inscrutable :)

Reply:
doing it all in node would be at least a dozen lines though :)